src/supabase/supabase_manager.py

```
@@ -587,17 +591,167 @@ def upload_all(self, tables: List[str] = None,
+        self._execute_sql(create_sql)
+
+        # Step 3: INSERT via Supabase client (batch upload)
+        records = self._df_to_records(df)
+        uploaded = 0
+        errors = []
+        for i in range(0, len(records), self.BATCH_SIZE):
+            batch = records[i:i + self.BATCH_SIZE]
+            try:
+                self.client.table(table_name).insert(batch).execute()
+                uploaded += len(batch)
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)